src/assets/data/character/defender.yml

```diff
@@ -267,10 +342,6 @@
       effects: 
         - desc: HP +20%
           all: true
-          meta:
```

Comment:
Don't remove these. These are important to the party creator.

Reply:
I think I started working on this while you were adding party creator so on my version meta wasn't there.

Reply:
Also don't approve it defender's aren't appearing on the preview

Reply:
Check the indentation of your notes. The notes block should be the same alignment as the rush block. It shouldn't be further in.